src/app/api/finances/sync/route.ts

```
@@ -0,0 +1,772 @@
+      return property.created_time || null;
+    case 'status':
+      return property.status?.name || null;
+    case 'url':
+      return property.url || null;
+    case 'checkbox':
+      return property.checkbox || false;
+    case 'relation':
+      return property.relation?.map((rel: any) => rel.id) || [];
+    case 'formula':
+      // Formula can return different types, check the result
+      const formulaResult = property.formula;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)